utils/vlog/tests/pb_type.xml

```diff
@@ -0,0 +1,34 @@
+<pb_type blif_model=".subckt test_pb" name="test_pb" num_pb="1">
```

Comment:
I'm pretty sure `blif_model` is not valid on objects which have inner `<pb_type>` objects.